gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c

```diff
@@ -988,7 +988,9 @@ _get_ip_address (void)
     }
     addrs = addrs->ifa_next;
   }
-  while ((addrs = addrs->ifa_next));
+
+  if (addrs)
```

Comment:
uh...plz wait for updating this.

Reply:
IMO this is unnecessary. addrs should be a list, not null to release it.
```
struct ifaddrs *addrs, *addr;

getifaddrs (&addrs);
addr = addrs;
while (addr) {
  ...
  addr = addr->ifa_next;
}

freeifaddrs (addrs);
```

Reply:
nnstreamer-edge.git has this error: https://github.com/nnstreamer/nnstreamer-edge/pull/47/files

Reply:
> IMO this is unnecessary. addrs should be a list, not null to release it.
> 
> ```
> struct ifaddrs *addrs, *addr;
> 
> getifaddrs (&addrs);
> addr = addrs;
> while (addr) {
>   ...
>   addr = addr->ifa_next;
> }
> 
> freeifaddrs (addrs);
> ```

Deleting it can be done at nnstreamer-edge.git.